### PR TITLE
Throw an exception if the session has started when setCookieParams is called

### PR DESCRIPTION
**Pull request by NaokiTsuchiya**
Resolve https://github.com/auraphp/Aura.Session/pull/67#discussion_r187972034 and add test.

### CODE REVIEW

File: src/Session.php [hunk 1]
```diff
@@ -480,6 +480,10 @@ public function getCacheLimiter()
      */
     public function setCookieParams(array $params)
     {
+        if ($this->isStarted()) {
+            return;
```
Comment (harikt) [inline src/Session.php]: Do you think we should throw an exception and raise an error than suppressing it

Comment (koriym) [reply src/Session.php]: If you need this return, it is likely that you are doing `setCookieParams` in the wrong place, and surely an exception is better than ignoring it.

Comment (NaokiTsuchiya) [reply src/Session.php]: I think so.

Comment (koriym) [reply src/Session.php]: @harikt Is the exception name something like `SessionAlreadyStartedException`?

**Review by koriym — APPROVED** [review]

File: src/Session.php (hunk 1 above)
Comment (harikt) [reply src/Session.php]: I think we should follow the Aura.Router convention. No Exception at end. Rest looks good to me.

**Pull request merged**